charts/app-config/values-integration.yaml

```
@@ -2504,6 +2504,18 @@ govukApplications:
     helmValues:
       arch: arm64
       dbMigrationEnabled: false
+      ingress:
+        enabled: true
+        annotations:
+          <<: [*alb-ingress-defaults]
+          alb.ingress.kubernetes.io/scheme: "internal"
+          alb.ingress.kubernetes.io/group.order: "172"
+          alb.ingress.kubernetes.io/conditions.{{ .Release.Name }}: >
+            [{"field": "host-header", "hostHeaderConfig": { "values": [
+                "search-api-v2.{{ .Values.publishingDomainSuffix }}"
```

Do we need to label it v2?

```suggestion
                "search-api.{{ .Values.publishingDomainSuffix }}"
```

---

Possibly not - just trying to differentiate from search-api-not-v2, and reckon it might get confusing